packages/pentaho/src/Canvas/CanvasContext/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./CanvasContext";
```

Comment:
let's please [stop using](https://tkdodo.eu/blog/please-stop-using-barrel-files) [unnecessary](https://marvinh.dev/blog/speeding-up-javascript-ecosystem-part-7/) [barrel](https://dev.to/tassiofront/barrel-files-and-why-you-should-stop-using-them-now-bc4) [files](https://dev.to/thepassle/barrel-files-a-case-study-o5p)